volmdlr/wires.py

```diff
@@ -3950,7 +3984,7 @@ def axial_symmetry(self, line):
 
     def discretization_points(self, *, number_points: int = None, angle_resolution: int = 20):
         if not number_points and angle_resolution:
-            number_points = math.ceil(volmdlr.TWO_PI * angle_resolution) + 2
+            number_points = math.ceil(3.1415 * angle_resolution) + 2
```

Comment:
why not using math.pi ?

Reply:
I thought in performance, but I dont really know if it makes considerable difference. I just thought that because we do math.ceil() we wouldn't have differences in approximating for 3.1415. But we can put math.pi for readability